packages/runtime-common/realm.ts

```diff
@@ -937,6 +829,9 @@ export class Realm {
     return undefined;
   }
 
+  // TODO we could really improve performance if this utilized the index instead
+  // of directly hitting the filesystem--especially the TS transpilation
+  // involved in making JS.
```

Comment:
is this a full project or could an issue be opened for it?

Reply:
I think it’s probably a project